micromasters/utils.py

```diff
@@ -16,3 +17,12 @@ def webpack_dev_server_url(request):
     Get the full URL where the webpack dev server should be running
     """
     return 'http://{}:{}'.format(webpack_dev_server_host(request), settings.WEBPACK_DEV_SERVER_PORT)
+
+
+def load_json_from_file(project_rel_filepath):
+    """
+    Loads JSON data from a file
+    """
+    path = '{}/{}'.format(settings.BASE_DIR, project_rel_filepath)
+    with open(path, 'r') as f:
+        return json.loads(f.read())
```

Comment:
I think you can do

```python
json.load(f)
```

here instead, I do very little python stuff these days so I don't know if there's a reason not to pass the file handle around.